maintainers/maintainer-list.nix

```diff
@@ -3422,6 +3422,10 @@
     githubId = 5241813;
     name = "Carter Sande";
   };
+  casaca = {
+    name = "J McNutt";
+    email = "jmacasac@hotmail.com";
```

Comment:
This won't work nowadays. You won't get pinged when there's someone opening issues or PRs

Reply:
OK, on it.